package.json

```diff
@@ -33,7 +33,7 @@
     "private::test.python": "python -m unittest tests/unit/format_test.py",
     "private::test.unit": "cypress run --browser chrome --spec 'tests/cypress/tests/unit/**/*'",
     "private::test.standalone": "cypress run --browser chrome --spec 'tests/cypress/tests/standalone/**/*'",
-    "build.watch": "webpack-dev-server --content-base dash_table --mode development --config webpack.dev.config.js",
+    "build.watch": "webpack-dev-server --disable-host-check --content-base dash_table --mode development --config webpack.dev.config.js",
```

Comment:
In Browserstack, when running on iOS devices, `localhost` is redirected to `bs-local` which webpack-dev-server doesn't like, this flag, as its name indicates, makes it not care